changelog/153.bugfix.rst

```diff
@@ -1 +1 @@
-Make it possible to use the ``requests`` library inside the default Rasa SDK container.
+Adding sys.exit(1) right after Import Error in register_package() in executer.py
```

Comment:
this should describe how the change affects the user instead of what the code implementation looks like. How about something like like
```suggestion
Exit program (``python -m rasa_sdk --actions actions`` or 
``rasa run actions --actions actions``) if a requested module under the 
``--actions`` command-line option cannot be found. 
```